Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version:4.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Cache",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "Cache",
+            targets: ["Cache"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(url: "https://github.com/onmyway133/SwiftHash.git", .upToNextMinor(from: "2.0.0")),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "Cache",
+            dependencies: ["SwiftHash"],
+            path: "Source/Shared"),
```

Comment:
Can we use `exclude` to exclude a directory ?

Reply:
Yeah I think that should work but you don't really want to exclude it if you installed it using package manager for other platforms right?

Reply:
For now, I've done the exclude unless someone has a better idea? But it would probably require restructuring the directories.